### ISSUE TITLE
OBS opening network connections on startup even though I disabled everything

### ISSUE THREAD

**Issue by AtmanActive** [obsproject/obs-studio#10928]
### Operating System Info

Windows 10

### OBS Studio Version

30.1.2

### OBS Studio Log URL

[rtmp-services plugin] Remote update of URL "https://obsproject.com/obs2_update/rtmp-services/v5/package.json" failed: Failed to connect to obsproject.com port 443 after 5 ms: Couldn't connect to server

### Expected Behavior

I wanted to use OBS for my private screen replay recording for work. I wanted to make sure my OBS instance is private and not streaming and not even connecting to internet. So I went to Settings and disabled automatic update check. I would expect OBS to not touch the network going forward.

### Current Behavior

OBS still tries to connect to 142.4.216.103:443 each and every time it is started.

### Steps to Reproduce

1. Install a personal firewall like Evorim FreeFirewall or Comodo Personal Firewall or install Wireshark or any other software that will show you when and how OBS is connecting to the internet
2. Download OBS v30.1.2 windows ZIP version, unpack to any folder
3. Start obs64.exe - observe dozens of internet connection attempts by OBS
4. Go to Settings and disable "Automatically check for updates on startup"
5. Exit OBS
6. Start OBS again - observe OBS still trying to connect to 142.4.216.103:443 three times each and every time it is started
...

### Anything else we should know?

This issue is irrelevant for streamers but it is crucial for private use of OBS for continuous screen recording for Replay purposes (bug reporting).

What is the goal here? The goal is to make OBS completely trustworthy by making sure it does NOT touch the internet when not told to.

Thanks.

**Comment by RytoEX** [comment]
This is probably a duplicate of #4044.

**Comment by Fenrirthviti** [comment]
Duplicate of #4044

**Comment by Fenrirthviti** [comment]
As noted in the other issue, the checkbox current disables the updates for OBS itself, it is not intended to, no does it claim to be, disabling all network communication at all.

I don't know that there's a significant amount of value in us spending time trying to offer it as an option, as the non-streaming network connectivity of OBS is extremely limited and easily blocked by the most basic of firewall rules, and should not impact the overall usability of the application.
